README.md

```diff
@@ -27,149 +28,171 @@ First, install the library via [npm](https://www.npmjs.com/get-npm):
-    pageSetId: 'My  Page Set', // the name of the feature in your features.json or the name of your template file
-    id: 90210, // the uid of the entity your page represents
-    name: 'static',
-  },
-  pagesReferrer: 'https://www.google.com', // e.g. document.referrer
-  path: '/foo/bar', // e.g. window.location.pathname
-  businessId: 12345, // this comes from the url of your Yext account
-  production: false, // set to true if you are in the production environment
-  siteId: 654321, // the id of your site, you can find this in the url of your deploy page
+pagesAnalytics.report({
+  action: "C_MY_CUSTOM_EVENT"
 });
 ```
-
-Now that the pages analytics reporter is initialized, we can fire a pageview:
+Additionally, you can send arbitrary conversion events by specifying a `value` JSON object with a dollar `amount` and a `currency` in ISO format. 
 ```ts
-pagesAnalytics.pageView();
+chatAnalytics.report({
+  action: "C_CONVERSION_EVENT"
+  value: {
+    amount: 10,
+    currency: "USD",
+  },
+});
 ```
+To learn more about sending conversion events, see our [API documentation](https://hitchhikers.yext.com/docs/eventsapis/events/events#operation/sendEvents).
 
-If a user clicks on a CTA, we can track a CTA Click Event
+#### Custom Properties
+You can attach custom properties to your analytics events by specifying either `customTags` or `customValues` with your request. `customTags` represent up to ten **string** key-value pairs and `customValues` represent up to ten **numeric** key-value pairs.
 
+For example, if I set up an `ORDER` event for my restaurant and wanted to track whether a promotional code was used on the order, I could add an `promoCode` custom tag to the event.
 ```ts
-import { CtaClick } from '@yext/analytics';
-
-pagesAnalytics.track(CtaClick);
+pagesAnalytics.report({
+  action: "C_CONVERSION_EVENT",
+  sessionId: "e790f75d-4f1e-4a1b-b57b-9a456019b176",
+  value: {
+    amount: 35.50,
+    currency: "USD",
+  },
+  customTags: {
+      "promoCode": "SPRING15OFF"
+  }
+});
 ```
-
-We can also fire an event on any other type of user interaction and give it a custom name:
+Additionally, if I wanted to record the discount amount of the promotion, I could add a `promoDiscount` custom value to the `ORDER` event.
 ```ts
-pagesAnalytics.track({eventType: 'C_MY_CUSTOM_EVENT'});
+pagesAnalytics.report({
+  action: "C_CONVERSION_EVENT",
+  sessionId: "e790f75d-4f1e-4a1b-b57b-9a456019b176",
+  value: {
+    amount: 35.50,
+    currency: "USD",
+  },
+  customTags: {
+      "promoCode": "SPRING15OFF"
+  },
+  customValues: {
+      "promoDiscount": 41.76
+  }
+});
 ```
+### Debugging
+We use `fetch()` + `keepalive` by default in [supported browsers](https://developer.mozilla.org/en-US/docs/Web/API/fetch) to make debugging easier. For browsers like Firefox that do not support `keepalive`, we use `fetch` only. Be warned, since `forceFetch` uses `fetch()` without `keepalive`, **requests in progress will be canceled if the page is unloaded**.
```

Comment:
This seems to indicate that we don't use beacon at all?

Reply:
Ah, I forgot chris edited it since we were still figuring out whether we will still use beacon